src/src/tramp.c

```diff
@@ -75,6 +74,8 @@ void trampBuildvPacket(void)
 
 void trampBuildsPacket(void)
 {
+    uint16_t temperature = 25; // Dummy value.
```

Comment:
Can you add a quick comment here why this cant be o for QS.

Reply:
maybe i can butt in: in the past i have checked for a non zero value here for "valid data was received"
however as the temperature was not used in any way i have since removed this check.

so this can now be null and probably should stay that way (?) to indicate that no valid temperature reading is available.

Reply:
That is fixed to QS now and value reverted back to 0.